gr-digital/include/gnuradio/digital/constellation_soft_decoder_cf.h

```diff
@@ -40,7 +40,7 @@ class DIGITAL_API constellation_soft_decoder_cf : virtual public sync_interpolat
      * 'constellation'. Use base() method to get a shared pointer to
      * this base class type.
      */
-    static sptr make(constellation_sptr constellation);
+    static sptr make(constellation_sptr constellation, float npwr);
```

Comment:
API change

Reply:
should line 43 include for instance "float npwr=1.0"?

Reply:
Yes. Anywhere a param is added in a public header, existing code needs to compile against it. Adding a default here would allow that.